test/live_server.rb

```diff
@@ -1,4 +1,5 @@
 require 'sinatra/base'
+load 'test/helper.rb'
```

Comment:
I'm guessing the ruby syntax warnings about re-declarations are due to this line, as some of the things it's complaining about are defined in `test/helper.rb`. I'm pretty sure this test helper script shouldn't try to include the setup for the test suite itself.

I'm not sure why you have this `load` here at all. Is it because you want to use `Faraday::TestCase.big_string`? If so, you can just copy that method over to `Faraday::LiveServer`. I think it's too small to worry about the duplication.